compare_output.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
```

Comment:
Some documentation could be useful here (how to use the script, what it does), or consider a `--help` output.